src/main/java/ti4/helpers/AgendaHelper.java

```diff
@@ -2296,9 +2298,7 @@ public static int[] getVoteTotal(Player player, Game game) {
     }
 
     public static List<Player> getVotingOrder(Game game) {
-        List<Player> orderList = new ArrayList<>(game.getPlayers().values().stream()
-            .filter(Player::isRealPlayer)
-            .toList());
+        List<Player> orderList = Helper.getSpeakerOrPriorityOrder(game);
```

Comment:
`getVotingOrder` is used in several places to manage queues and such. It seems like a bug in the "mandatory pre-vote" code sometimes breaks out of mandatory pre-voting. When that happens, this prevents further issues and confusion.